<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Equalizer</class>
 <widget class="QDialog" name="Equalizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equalizer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Brightness</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="brightnessSlider">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="brightnessLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Contrast</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="contrastSlider">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="constrastLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Saturation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="saturationSlider">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="saturationLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Gamma</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="gammaSlider">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="gammaLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Hue</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSlider" name="hueSlider">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="hueLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>brightnessSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>brightnessLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contrastSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>constrastLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saturationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>saturationLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gammaSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>gammaLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hueSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hueLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
